source/dofs/dof_tools_sparsity.cc

```diff
@@ -1018,18 +1018,16 @@ namespace DoFTools
                                                        keep_constrained_dofs,
                                                        bool_int_dof_mask[cell->active_fe_index()]);
               // Loop over all interior neighbors
-              for (unsigned int face = 0;
-                   face < GeometryInfo<dim>::faces_per_cell;
-                   ++face)
+              for (unsigned int face_n = 0;
+                   face_n < GeometryInfo<dim>::faces_per_cell;
+                   ++face_n)
```

Comment:
It's not wrong to change this, but I will note that we pretty uniformly use `face` as the index when looping over faces.